tests/test_cloudpath_file_io.py

```diff
@@ -96,6 +96,16 @@ def test_file_read_writes(rig, tmp_path):
     assert cloud_rel_paths == dled_rel_paths
 
 
+def test_cloud_path_download_to(rig, tmp_path):
+    p = rig.create_cloud_path("dir_0/file0_0.txt")
+    dl_dir = tmp_path
+    assert not dl_dir.joinpath(p.name).exists()
+    print(list(dl_dir.iterdir()))
+    p.download_to(dl_dir)
+    print(list(dl_dir.iterdir()))
+    assert dl_dir.joinpath(p.name).exists() & dl_dir.joinpath(p.name).is_file()
```

Comment:
```suggestion
    assert (dl_dir / p.name).is_file()
```

`is_file` will evaluate to False if the path doesn't exist.